index.rst

```diff
@@ -0,0 +1,20 @@
+FRED Language Tutorials
+=======================
+
+Welcome to the FRED tutorials documentation.
+
+FRED is a programming language and modeling platform for simulating how a population changes over time. This document
+discusses the tutorials provided for the FRED Modeling Language\ |trade| to provide examples and guidance for FRED users.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Contents:
+
+   Introduction <README>
+   Simple Flu <simpleflu/README>
+   Flu with Behavior <flu-with-behavior/README>
+   Flu with School Closure <school-closure/README>
+   Flu with Vaccine <vaccine/README>
```

Comment:
Does this syntax result in something like, e.g. `<a href="simpleflu/README.html">Simple Flu</a>`? Previously I've just specified the path to the source files under the `toctree` directive, in which case the link text is taken from the top level header of the target document. It would be good to know if alternative link texts can be specified in this way, however!